src/main/java/teammates/storage/entity/AdminEmail.java

```diff
@@ -63,6 +62,19 @@ public AdminEmail(List<String> addressReceiver, List<String> groupReceiver, Stri
         this.isInTrashBin = false;
     }
 
+    // This constructor is used only for AdminEmailAttributesTest#testValueOf()
+    public AdminEmail(Long emailId, List<String> addressReceiver, List<String> groupReceiver, String subject,
```

Comment:
It's not good to add new public constructor only for one test purpose. Instead, try to avoid null pointer exception in getEmailId() method:
```java
    public String getEmailId() {
        return emailId == null ? null : Key.create(AdminEmail.class, this.emailId).toWebSafeString();
    }
```